app/src/main/java/com/norvellium/tasky/presentation/login/LoginState.kt

```diff
@@ -0,0 +1,9 @@
+package com.norvellium.tasky.presentation.login
+
+data class LoginState(
+    val email: String = "",
+    val password: String = "",
+    val isValidEmail: Boolean = true,
+    val isValidPassword: Boolean = true,
+    val errorMessage: String = "",
```

Comment:
Talked about this in the last group call, I'd recommend to not set error messages to an empty string if it's not existent. Instead make it nullable, since that's exactly what nulls are used for. If you get an error message which is empty you might not be sure if there is an error or not (just because there's no message doesn't mean there's no error)